### PR TITLE
Include owner, offer, and lease information in node list

### PR DESCRIPTION
**Pull request by betaredex**

### CODE REVIEW

**Comment by tzumainn** [comment]
I think there may need to be adjustments to our CI, but try running 'tox -epy3' ; that shows that the tests are not actually passing.

File: esileapclient/tests/unit/osc/v1/test_node.py [hunk 1]
```diff
@@ -38,9 +38,44 @@ def test_node_list(self):
...
 
         self.assertEqual(collist, list(columns))
 
-        datalist = ((fakes.node_name,),)
+        datalist = ((fakes.node_name,
+                     fakes.node_owner
+                     ),)
+        self.assertEqual(datalist, tuple(data))
+
+    def test_lease_list_long(self):
```
Comment (tzumainn) [inline esileapclient/tests/unit/osc/v1/test_node.py:54]: This test is probably misnamed.

File: esileapclient/tests/unit/osc/v1/test_node.py [hunk 2]
```diff
@@ -38,9 +38,44 @@ def test_node_list(self):
...
+            "Lease UUID",
+            "Lessee"
         ]
 
         self.assertEqual(collist, list(columns))
 
-        datalist = ((fakes.node_name,),)
+        datalist = ((fakes.node_name,
+                     fakes.node_owner
+                     ),)
```
Comment (tzumainn) [inline esileapclient/tests/unit/osc/v1/test_node.py:51]: I think you'll end up with empty strings for the additional fields here.

File: esileapclient/v1/node.py [hunk 3]
```diff
@@ -21,14 +21,24 @@
...
-        'name': "Name"
+        'name': "Name",
+        'owner': "Owner",
+        'offer_uuid': "Offer UUID",
+        'lease_uuid': "Lease UUID",
+        'lessee': "Lessee"
     }
 
-    _creation_attributes = ['name']
+    _creation_attributes = ['name', 'uuid', 'owner', 'offer_uuid',
```
Comment (tzumainn) [inline esileapclient/v1/node.py:40]: I'm not sure this variable is needed at all, since we'll never be creating nodes from the esi-leap CLI

**Review by tzumainn — CHANGES_REQUESTED** [review]
Pretty good! A few test failures to fix.

File: esileapclient/v1/node.py (hunk 3 above)
Comment (betaredex) [reply esileapclient/v1/node.py:40]: I thought so, too, but it threw an error when I tried to remove it. I might take another look.

File: esileapclient/tests/unit/osc/v1/test_node.py (hunk 1 above)
Comment (betaredex) [reply esileapclient/tests/unit/osc/v1/test_node.py:54]: Yup, missed that one

File: esileapclient/tests/unit/osc/v1/test_node.py (hunk 2 above)
Comment (betaredex) [reply esileapclient/tests/unit/osc/v1/test_node.py:51]: Fixed

File: esileapclient/v1/node.py (hunk 3 above)
Comment (betaredex) [reply esileapclient/v1/node.py:40]: Yeah, if I take it out, it throws ```Can't instantiate abstract class Node with abstract methods _creation_attributes```

**Review by tzumainn — APPROVED** [review]

**Comment by tzumainn** [comment]
Looks good!

**Pull request merged**